dvtemplates/fedora_new_image.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+error_message() {
+    if [ $1 -eq 1 ]; then
+        echo -e "Unable to connect to the Fedora repository"
+    elif [ $1 -eq 2 ]; then
+        echo -e "Invalid Image Version"
+    elif [ $1 -eq 3 ]; then
+        echo -e "No Container Image found in the registry"
+    elif [ $1 -eq 4 ]; then
+        echo -e "Container Image for the Latest Fedora version is already present"
+    fi
+    exit
+}
+
+trap 'error_message $?' EXIT
+
+#TODO : change it to https://quay.io/containerdisks/fedora_images ?
```

Comment:
I like the idea of having an org created called containerdisks, and have a repository created for every OS Image. The tag associated with each image could be the version of the Image. Any thoughts?

Reply:
I thought about `quay.io/kubevirt/containerdisk:fedora` or something like that, but I have no hard objections for a new org

Reply:
Chose this as the final repo - "quay.io/kubevirt/fedora-images"